docs/core/diagnostics/debug-memory-leak.md

```diff
@@ -132,6 +132,9 @@ Once the dump is collected, you should have sufficient information to diagnose t
 
 In this tutorial, you're now done with the [Sample debug target](/samples/dotnet/samples/diagnostic-scenarios/) and you can close it. Navigate to the terminal that started the server, and press <kbd>Ctrl+C</kbd>.
 
+> [!NOTE]
+> After restarting the failed process and letting it run for around one minute, you could collect another dump in the same way explained above. You would then have two dumps over a period of time, and could compare them to assist in seeing where the memory is growing.
```

Comment:
Collecting a dump after restart won't be very useful because restarting has reset all memory usage. For a more useful comparison over time they should let the original process continue running after collecting the first dump and collect the 2nd one prior to restarting. This NOTE should probably go at the end of the Generate Memory dump section.